src/styles/theme.scss

```diff
@@ -1,19 +1,36 @@
+/* ------------------------------
+
+  seAP Fonts
+
+------------------------------ */
+
+@font-face {
+font-family: 'Frutiger LT Light';
+font-style: normal;
+font-weight: normal;
+src: url('./fonts/Frutiger LT 45 Light.woff') format('woff');
+}
+
+@font-face {
+font-family: 'Frutiger LT Bold';
+font-style: normal;
+font-weight: normal;
```

Comment:
Should this be bold?

Reply:
no, because the font is bold by default so that's its normal weight. I got this from the font file.

Reply:
You can do it that way but would have to specify font-family for instances of bold. The name of the font-family should be the same for both (Frutiger) and you can vary the styles as usual in the css without declaring a new family

For more detailed explanation see:
https://css-tricks.com/whats-deal-declaring-font-properties-font-face/ and https://www.456bereastreet.com/archive/201012/font-face_tip_define_font-weight_and_font-style_to_keep_your_css_simple

Reply:
I'll wait for the files from seAp - Rose mentioned it in Slack that they are sending these - and will change then.

Reply:
I guess don't bother waiting but change them both to Friutiger family so we have one family to name.